bridge/contracts/utils/ImmutableAuth.sol

```diff
@@ -289,6 +343,87 @@ abstract contract ImmutableLiquidityProviderStaking is ImmutableFactory {
     }
 }
 
+abstract contract ImmutableLocalERC1155BridgePoolV1 is ImmutableFactory {
```

Comment:
change this to ImmutableNativeERC1155BridgePoolV1

Reply:
Actually removed, belong in bridge-pool-base